payments-core/api/payments-core.api

```diff
@@ -809,6 +809,7 @@ public final class com/stripe/android/Stripe$Companion {
 }
 
```

Comment:
I think we want to make all of these internal functions, so there is no change to the api until we are ready for launch